core/src/main/java/org/elasticsearch/common/http/client/HttpDownloadHelper.java

```diff
@@ -378,9 +369,6 @@ private URLConnection openConnection(URL aSource) throws IOException {
                         responseCode == HttpURLConnection.HTTP_MOVED_TEMP ||
                         responseCode == HttpURLConnection.HTTP_SEE_OTHER) {
                     String newLocation = httpConnection.getHeaderField("Location");
-                    String message = aSource
```

Comment:
@s1monw I removed also this because it's not used in our code. It's unrelated to the change though.